packages/widgets/widget-utils/src/bootstrap/index.js

```diff
@@ -1,15 +1,23 @@
 import bootstrapWidgets from './bootstrapWidgets'
 import {getBackendUrl, loadScriptAsync} from './utils'
 ;(() => {
-  const backendUrl = getBackendUrl(document)
+  let backendUrl, assetUrl
+  if (window.backendUrl) {
+    backendUrl = window.backendUrl
```

Comment:
could we rename this global variable with `tocco` prefix: `toccoBackendUrl`?

Reply:
And maybe add it to the `constants.js` so that we have all global variables in one place?